src/test/java/org/apache/maven/shared/utils/cli/CommandLineUtilsTest.java

```diff
@@ -19,22 +19,29 @@
  * under the License.
  */
 
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Locale;
 import java.util.Map;
 import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
 import org.apache.maven.shared.utils.Os;
 
-import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
Would please so nice to keep explicit import instead of asteric imports...Thanks.

Reply:
Done!  Thanks for catching it.